.github/workflows/ci.yml

```diff
@@ -157,6 +157,54 @@ jobs:
         source pygraphistry/bin/activate
         ./bin/test-umap-learn-core.sh
 
+
+  test-gpu-umap:  # well cpu until get a github actions gpu node
```

Comment:
this might work now? https://medium.com/@tajinder.singh1985/exploring-github-nvidia-powered-gpu-hosted-runner-32b172a92c7e

Reply:
maybe we should add some sort of test to confirm the gpu is enabled + gpu is used, e.g.,

```bash
nvidia-smi || exit 1

python3 -c "import cudf; cudf.DataFrame({'x': [0,1,2]})['x'].sum()"
```

Reply:
if that is overreach for this pr, we should comment this out